go/governance/api/sanity_check.go

```diff
@@ -0,0 +1,146 @@
+package api
+
+import (
+	"fmt"
+	"sort"
+
+	epochtime "github.com/oasisprotocol/oasis-core/go/epochtime/api"
+	upgrade "github.com/oasisprotocol/oasis-core/go/upgrade/api"
+)
+
+// SanityCheck performs a sanity check on the consensus parameters.
+func (p *ConsensusParameters) SanityCheck() error {
+	if !p.MinProposalDeposit.IsValid() {
+		return fmt.Errorf("min_proposal_deposit has invalid value")
+	}
+	// Quorum must be less than 100.
```

Comment:
Or equal to? The check is fine, just the comment needs to be fixed. (Also below.)

Reply:
Fixed